summary_workflows/quantification/main.nf

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env nextflow
+
+if (params.help) {
+	
+	log.info"""
+	    =====================================================
+	    APAeval QUANTIFICATION BENCHMARKING PIPELINE
+	    Author(s): Yuk Kei Wan (*1,*2), Asier Gonzalez (*3), CJ Herrmann (*4)
+	    *1 iRNA COSI
+	    *2 Genomic Institute of Singapore, A*STAR, Singapore
+	    *3 Barcelona Supercomputing Center, Barcelone, Spain
+	    *4 Biozentrum, University of Basel, Switzerland
+	    =====================================================
+	    Usage:
+	    Run the pipeline with default parameters read from nextflow.config:
+	    nextflow run main.nf -profile docker
+	    Run with user parameters:
+	   nextflow run main.nf -profile docker --input {execution.wf.APA.prediction.file} --participant_id {tool.name} --goldstandard_dir {gold.standards.dir} --challenge_ids {analyzed.challenges} --assess_dir {benchmark.data.dir} --results_dir {output.dir}
```

Comment:
`results_dir` is not described in the mandatoryy arguments sections. 
What is the difference of `assess_dir`and this results directory? Seems to save similar output.